main/lib/sessionApi.js

```diff
@@ -1,11 +1,9 @@
-export function setSession(data) {
-  sessionStorage.setItem('accessToken', data.access_token)
-}
-
-export function getSession() {
-  return sessionStorage.getItem('accessToken')
-}
+const getSession = () => sessionStorage.getItem('accessToken')
```

Comment:
This format just seems cleaner to me, and it removes multiple `export` calls. I intentionally left braces around the bodies of the two other methods since I didn't want to deal with any weird side effects from the implicit `return`.